### DISCUSSION TITLE
Media sync meeting 15 November 2023

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/media#68] [General]
We have access! Socials strategy
What is good to share, examples of what is, and is not appropriate.

Blogpost on new chairs - need acronyms extended, include blurb for Jesse.
Where to publish?
 https://hackmd.io/@indiebio/OMI_chairs_press_release

General discussion

Previous meeting: https://github.com/omigroup/media/discussions/67

**Comment by indiebio** [comment]
e.g. youtube, blog, once a week at least.
Mastodon, socials, try for more often (e.g. once a day) but commit to at least once a week.

A board of sharing - post in Discord #omi-media 

ping antpb for blogpost (website). Change Twitter to Mastodon.

**Comment by indiebio** [comment]
From community meeting, what people would want to be improved:
-  Visibility / prioritization of work items for the standards efforts
- Want to see events accelerate. More emphasis. More action. Events and OMI can be a connecting fibre between the different groups and the general public.
- Clearer navigation, website, github, discord needs a revamp - things that get used more often needs more emphasis, archive out of date things, just a bit of maintenance.
- more working groups? with regular meetings. virtual world museum, digital communities/ux
- General intro pages for different working groups on website - glTF, roderick's protocol, for a start.
- energise people, if people do stuff, tell other people about it.
- We need more! MOAR! More people gettting involved, more standards.
- More participation, focus, structure to build systems that encourage people to work together. Having conversations, telling stories.
- Want to connect, need more support. 
- present file formats and protocols in a way that facilitates independent publishing of media

**Comment by jimmy6DOF** [comment]
OMi Socials Should promote and or get involved someway in the [Metaverse Safety Week Dec10-15](https://metaversesafetyweek.org/)
 - [WebXR Summit Series](https://www.youtube.com/watch?v=uWrD695IJ7I)
 - [Twitter Post](https://x.com/MetaSafetyWeek/status/1722322313935462877?s=20)

**Comment by jimmy6DOF** [comment]
and this example of a glTF in the wild doing it's thing : 
https://x.com/MattBCool/status/1725551800147915114?s=20
